README.md

```diff
@@ -117,6 +117,7 @@ To generate the Flinkster GBFS feed, you need to provide the following environme
 
 To generate a GBFS feed for a Free2move location, you need to provide the following environment variables:
 
+* `FREE2MOVE_BASE_URL=<FREE2MOVE_BASE_URL>`
```

Comment:
I think we should document that this URL must not have a trailing slash.

Reply:
I added stripping a trailing slash in [c7829ca](https://github.com/mobidata-bw/x2gbfs/pull/203/commits/c7829ca910f9c0a82c05e0783634c5da03a33e55), so it will not matter